libraries/standard/mqtt/src/iot_mqtt_subscription.c

```diff
@@ -59,7 +59,7 @@ typedef struct _topicMatchParams
 typedef struct _packetMatchParams
 {
     uint16_t packetIdentifier; /**< Packet identifier to match. */
-    int32_t order;             /**< Order to match. Set to `-1` to ignore. */
+    int32_t order;             /**< Order to match. Set to `MQTT_REMOVE_ALL_SUBSCRIPTIONS` to ignore. */
```

Comment:
Since MQTT_REMOVE_ALL_SUBSCRIPTIONS is a documented macro, it should be referred to as `#MQTT_REMOVE_ALL_SUBSCRIPTIONS` in doxygen comments

Reply:
Changed